.storybook/preview.tsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect } from 'react'
+import type { Preview } from '@storybook/react'
+import type { StoryFn } from '@storybook/react'
+
+import '@open-inwoner/design-tokens/dist/css/index.css'
+// Use this ↓ for OIP components and NLDS components thst we have set values for
+// import '../src/open_inwoner/static/bundles/open_inwoner-css.css';
+
+const withThemeClass = (Story: StoryFn) => {
+  useEffect(() => {
+    document.body.classList.add('openinwoner-theme')
+    return () => {
+      document.body.classList.remove('openinwoner-theme')
+    }
+  }, [])
```

Comment:
Why not just add the class or check if the class exists and add it (this useEffect is not really needed).
```suggestion
    document.body.classList.remove('openinwoner-theme')
```

Reply:
Yes, but the class needs to be added, not removed.

Reply:
Oops i meant that wrong suggestion... It should have been
```suggestion
    document.body.classList.add('openinwoner-theme')
```